src/actions/goto-anywhere.js

```diff
@@ -1,6 +1,6 @@
 import type { ThunkArgs } from "./types";
 
-export function setSourceSearchQuery(queryString: string) {
+export function setGotoAnywhereQuery(queryString: string) {
```

Comment:
lol.. not sure about `gotoanywhere` what does atom/code call this modal?

Reply:
Sublime called it [`GotoAnything`](http://docs.sublimetext.info/en/latest/file_management/file_navigation.html#goto-anything), Atom doesn't specifically call it out by a name in their Flight Manual.